src/index.html

```diff
@@ -2,11 +2,423 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="value">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="Home" class="logo"><img src="/images/logo.png" alt="#"></a>
+      <nav class="nav">
+        <ul class="nav__menu">
+          <li class="nav__list">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__list">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__list">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__list">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__list">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__list">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    
+    <main class="catalog">
+      <div 
+        class="card" 
+        data-qa="card-hover"
+      >
+        <img 
+          class="card__img-imac"
+          src="./images/imac.jpeg" 
+          alt="#"
+        >
+        <div class="card__info">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__product-code">
+          Product code: 195434
+        </div>
+        <div class="card__review">
+          <div class="star stars--4">
+            <div class="star__stars"></div>
```

Comment:
```suggestion
            <div class="stars stars__star"></div>
```
`stars` is BEM block (that has modifcators such as `stars--4`)
`star` is an element fo `stars` block
overwrite your CSS according to changes in html

Reply:
> `stars` is BEM block (that has modifcators such as `stars--4`) `star` is an element fo `stars` block overwrite your CSS according to changes in html

<img width="701" alt="image" src="https://user-images.githubusercontent.com/93718680/200301652-2305c375-bc60-4d32-bbc3-a6269f6379aa.png">